Alignment/CommonAlignmentProducer/python/ALCARECOPromptCalibProdSiPixelAliHLT_cff.py

```diff
@@ -13,11 +13,9 @@
 
 # Ingredient: onlineBeamSpot
 import RecoVertex.BeamSpotProducer.BeamSpotOnline_cfi
-onlineBeamSpot = RecoVertex.BeamSpotProducer.BeamSpotOnline_cfi.onlineBeamSpotProducer.clone()
-
-import RecoVertex.BeamSpotProducer.onlineBeamSpotESProducer_cfi as _mod
-BeamSpotESProducer = _mod.onlineBeamSpotESProducer.clone(
-    timeThreshold = 999999 # for express allow >48h old payloads for replays. DO NOT CHANGE
+onlineBeamSpot = RecoVertex.BeamSpotProducer.BeamSpotOnline_cfi.onlineBeamSpotProducer.clone(
+    useBSOnlineRecords = True,
+    timeThreshold = 999999
```

Comment:
Do you think it may be worth repeating the previous comment ("for express" etc.)?